tests/devices/default_qubit/test_default_qubit.py

```diff
@@ -1681,13 +1681,15 @@ def test_postselection_valid_analytic(self, param, mp, interface, use_jit):
         param = qml.math.asarray(param, like=interface)
 
         @qml.qnode(dev, interface=interface)
+        @qml.defer_measurements
```

Comment:
Mind switching the order of the qnode and the transform?

```
        @qml.defer_measurements
        @qml.qnode(dev, interface=interface)
```

While both ways are valid, applying it to the qnode is easier to track, has more features, and reduces uses of queueing.  Doesn't matter as much here,  but I'd like to have it as the de facto way of applying transforms in the codebase.